.gitignore

```diff
@@ -34,6 +34,9 @@ mergelog.txt
 *.user
 *.sln.docstates
 
+# NT.cmd
+nt.cmd
```

Comment:
Don't think you want to include this?

Reply:
I kinda do, honestly. Seeing it in the untracked files list is just tedious. If we end up with a version we want to commit later, it's easy enough to remove.

Reply:
Hm. Looks like I can [set up a local exclude list](https://stackoverflow.com/questions/1753070/how-do-i-configure-git-to-ignore-some-files-locally)
If that works, I don't need to pollute the shared file.

Reply:
removed this change.